app/views/schools/mentors/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_data(
+  title: Teachers::Name.new(@teacher).full_name,
+  caption: "TRN: #{@teacher.trn}",
+  header: Teachers::Name.new(@teacher).full_name,
+  backlink_href: schools_mentors_home_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m">Mentor details</h2>
+      <%= govuk_summary_list do |summary_list|
```

Comment:
I was wondering if throwing this in the `mentor_helper` and reusing the other helpers for name would be better? I'd definitely leverage `teacher_full_name` now though like you've done in the component.

Reply:
Not sure I follow 🤔

But just to clarify - this won't be merged in this PR. I only cherry-picked it from #464, and it’s already been refactored into a component further down the line.

I mainly needed the route here so the view wouldn't break while this PR was being reviewed - I admit I was a bit lazy 😅 I did leave a note about it in the PR description, not sure if you saw it.

Reply:
Gotcha, I didn't realise I was ignoring this too.